package.json

```diff
@@ -70,7 +70,8 @@
   },
   "files": [
     "dist",
-    "bin"
+    "bin",
+    "blog_example"
```

Comment:
I do not want to include blog_example. Instead, can you create an init directory and in there put in the files that we will want to start with?

Reply:
I copied the blog_example to init folder. In the future we can put new files or create a new demo init blog.